units/src/amount.rs

```diff
@@ -1835,6 +1851,40 @@ mod tests {
     #[test]
     fn from_int_btc_panic() { Amount::from_int_btc(u64::MAX); }
 
+    #[test]
+    fn test_signed_amount_try_from_amount() {
+        let ua_positive = Amount::from_sat(123);
+        let sa_positive = SignedAmount::try_from(ua_positive).unwrap();
+        assert_eq!(sa_positive, SignedAmount(123));
+
+        let ua_max = Amount::MAX;
+        let result = SignedAmount::try_from(ua_max);
+        assert_eq!(
+            result,
+            Err(OutOfRangeError {
+                is_signed: true,
+                is_greater_than_max: true
+            })
+        );
+    }
+
+    #[test]
+    fn test_amount_try_from_signed_amount() {
+        let sa_positive = SignedAmount(123);
+        let ua_positive = Amount::try_from(sa_positive).unwrap();
+        assert_eq!(ua_positive, Amount::from_sat(123));
+
+        let sa_negative = SignedAmount(-123);
+        let result = Amount::try_from(sa_negative);
+        assert_eq!(
+            result,
+            Err(OutOfRangeError {
+                is_signed: false,
+                is_greater_than_max: false
+            })
+        );
+    }
+    
```

Comment:
In 668b11f8ae815e521be13ba243eaa75aaa828c90:

Trailing whitespace.

Reply:
Removed it.